modules/460-log-shipper/docs/EXAMPLES.md

```diff
@@ -304,6 +304,28 @@ spec:
     endpoint: logstash.default:12345
 ```
 
+## Syslog
+
+The following examples sets severity for the syslog messages and uses the socket destination:
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: ClusterLogDestination
+metadata:
+  name: rsyslog
+spec:
+  type: Socket
+  socket:
+    mode: TCP
+    address: 192.168.0.1:3000
+    encoding: 
+      codec: Syslog
+  extraLabels:
+    syslog.severity: "alert"
+    # the request_id field should be present in the log message
+    syslog.message_id: "{{ request_id }}"
```

Comment:
Don't we need some validation/defaults for this?

Reply:
There is a default '-'